### PR TITLE
Emma/account creation footer

### PR DESCRIPTION
**Pull request by emmayuan25**

### CODE REVIEW

**Review by ldb297 — CHANGES_REQUESTED** [review]
Please change all exact pixel measurements to incorporate either view height, view width. 

maybe for font its okay to leave in exact pixel dimensions, but would recommend moving toward "em" and "rem" in the future. 

like the use of @media for widths, but where did you find these measurements? what is it based on? is this for phone screens, portrait tablets, and landscape screens?

overall, good work just make change for exact pixel dimensions

**Review by hs-credit-dev — CHANGES_REQUESTED** [review]
Screen Shot 2022-06-26 at 4 10 27 PM

also where are you incorporating the footer? i had to add it into app.js myself to test. 

this footer should carry across all screens right? so i would add it into the main app.js where it will eventually live. 

and attached is screenshot of what it looks like on my machine, it doesn't stretch across entire bottom of screen.

**Review by hs-credit-dev — APPROVED** [review]
nice work

**Review by ldb297 — APPROVED** [review]
great work

**Comment by BrownMY** [comment]
lets import the footer into the app

File: src/styles/footer.account.creation.styles.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    align-items: center;
+    bottom: 3em;
+    display: flex;
+    font-family: "Montserrat";
+    font-size: 0.5em;
+    font-style: SemiBold;
+    font-weight: 600;
+    height: 3em;
+    position: absolute;
+    width: 100%;
```
Comment (crlimacastro) [inline src/styles/footer.account.creation.styles.css:11]: Might want 100vw to ensure footer always stretches across screen

**Review by crlimacastro — APPROVED** [review]

**Review by BrownMY — DISMISSED** [review]

**Pull request merged**